Jenkinsfile

```diff
@@ -11,7 +11,7 @@ rocprimCI:
 
     def rocprim = new rocProject('rocprim')
 
-    def nodes = new dockerNodes(['gfx803', 'gfx900', 'gfx906'], rocprim)
+    def nodes = new dockerNodes(['gfx803 && ubuntu', 'gfx900 && ubuntu', 'gfx906 && ubuntu', 'gfx900 && centos7', 'gfx803 && ubuntu && hip-clang', 'gfx900 && ubuntu && hip-clang', 'gfx906 && ubuntu && hip-clang'], rocprim)
```

Comment:
multiline?

Reply:
Probably should be. Going to make another commit